Kickstarter-iOS/Views/Controllers/LoginViewController.swift

```diff
@@ -91,6 +106,26 @@ internal final class LoginViewController: UIViewController {
     self.passwordTextField.rac.text = self.viewModel.outputs.passwordText
     self.onePasswordButton.rac.hidden = self.viewModel.outputs.onePasswordButtonHidden
 
+    self.viewModel.outputs.showHidePasswordButtonToggled
+    .observeForUI()
+    .observeValues { [weak self] shouldShow in
+      guard let sself = self else { return }
```

Comment:
Here I would rename `sself` to `_self` since that's the name we usually use to check if `self` is not nil.

Reply:
For readability I'd move all of this to a private function in the file too 👍